### ISSUE TITLE
Redstone Ore generation?

### ISSUE THREAD

**Issue by garlicinyoureye** [AIE-Guild/AIE-Nomads-Modpack#66]
I saw a previous post about this and the code was posted indicating that redstone ore isn't biome specific and is set to generate between y levels of 0-16 with a variance of 16.  Has this been tested?  I ask because I've been mining for a solid hour-hour and half in sunflower plains, river, beach and swamp biomes and have managed to get a stack of diamonds and not a single redstone ore.  I may just be the unluckiest person ever but I thought the code for diamond generation was much rarer than the redstone code.

Thanks

**Comment by mrogaski** [comment]
Distribution was set to uniform with variance arguments, effectively disabling generation.  Set distribution to normal.

**Comment by garlicinyoureye** [comment]
Thanks for addressing this so quickly, I'm really enjoying the pack.  Can I change this in the file or do I need to wait until the update (if so can you tell me which file I will need to change)?  I'm assuming if I can change this that it will only effect chunks that haven' t been loaded?

Thanks again!

**Comment by garlicinyoureye** [comment]
Nevermind this question I downloaded the 1.0.5 version and updated my files.

**Comment by mrogaski** [comment]
Thank you for letting us know.  

You can regen the ore that's still hidden in chunks that have already been loaded by editing config/cofh/core/common.cfg and setting "B:RetroactiveOreGeneration=true".  Then reload the world, wait for the regeneration to finish (keep your console open), and set the option back to false before restarting again.  We did it on our server last night with a 3.5 GB world.  It took a while, but the world was playable.

**Comment by garlicinyoureye** [comment]
Luckily, I had just started the game and my world wasn't too big.  I've since ventured out to unloaded chunks and have no problem finding the ore.  I'm surprised someone from your server hadn't noticed this yet.

I really enjoy this pack.  I have been heavily playing Magic Farm 3 for the past few months and love the challenge it presents but felt the ability to progress wasn't hindered whatsoever.  

I have never played w/ the railcraft, or projectRed mod so the new item types are taking some getting to used to.  I do LOVE the scrap piles however, finding a sachel in one of those early game is like Christmas morning when gathering resources.

Is there any tick adjustment for crops that are planted on humus rather than soil?  I haven't noticed much of a difference in my game and didn't know if this was something that should/could be adjusted.   Overall, I do like the slower tick rate on the growth with a higher yield at maturity.  Also, I'm assuming the slower eating motion for larger saturation meals rather than smaller saturation meals was done on purpose?  I do like that addition whether if it was done intentionally or not.
